### DISCUSSION TITLE
style最新版本4.7.6没有进度和高能进度条选项

### DISCUSSION THREAD

**Discussion by whyrusoshy** [MuiseDestiny/zotero-style#1105] [Q&A]
如题，如何解决呀？我看前几个版本已经修复了这个问题的，怎么最新版本又没有了

**Comment by whyrusoshy** [comment]
旧版本也无法安装

**Comment by Lynsir** [comment]
遇到同样问题，前天安装的版本还有，昨天今天可能自动更新了就没有了

**Reply by whyrusoshy** [reply]
依然没有进度/高能进度选项，但是阅读文献后文章标题有高能进度条的显示结果了。以前读过的文章依然没有显示，只有新读的文献才显示进度。

**Reply by liudaye1** [reply]
我也是这样，请问解决了么

**Comment by liudaye1** [comment]
我也出现了这个问题，吃个饭回来，发现进度条都么了。addon item也没有删掉

**Comment by liudaye1** [comment]
是因为插件刚刚更新自动更新到4.8.5,数据丢失么

**Comment by Lynsir** [comment]
我现在新版本又好了 没问题了





发自我的iPhone


------------------ 原始邮件 ------------------
发件人: liudaye1 ***@***.***&gt;
发送时间: 2024年9月8日 23:14
收件人: MuiseDestiny/zotero-style ***@***.***&gt;
抄送: Lynsir ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [MuiseDestiny/zotero-style] style最新版本4.7.6没有进度和高能进度条选项 (Discussion #1105)





 
我也是这样，请问解决了么

**Reply by liudaye1** [reply]
之前的阅读进度也找回来了么
